srcpkgs/eslauncher2/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'eslauncher2'
+pkgname=eslauncher2
+version=0.8.10
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="alsa-lib-devel"
+short_desc="Next generation Endless Sky launcher"
+maintainer="xolophreny <xolophreny@proton.me>"
+license="GPL-3.0-only"
+homepage="https://github.com/EndlessSkyCommunity/ESLauncher2"
+distfiles="https://github.com/EndlessSkyCommunity/ESLauncher2/archive/refs/tags/v${version}.tar.gz"
+checksum=64180ada21b5a6df3fb72e41cc1bfb32e96c1b7f6a2d954acbbe7500dd5943d0
+
+if [ "$XBPS_TARGET_LIBC" == "musl" ] ; then
+	hostmakedepends+=" protobuf"
```

Comment:
This is weird why is this only required with mustl?

Reply:
Because one of the cargo crates this uses, `prost`, with itself ships a pre-built `protoc` binary that it needs, and that works on glibc but, despite them providing musl binaries, fails on musl. So that replaces it with the packaged version. Should it be put in regular `hostmakedepends` so it doesn't look as confusing?

Reply:
Yes the build probably shouldn't use binaries if possible.